typescript/graphql-sdl-first/src/server.ts

```diff
@@ -1,9 +1,11 @@
-import { GraphQLServer } from 'graphql-yoga'
+import { ApolloServer } from 'apollo-server'
 import { schema } from './schema'
 import { createContext } from './context'
 
-new GraphQLServer({ schema, context: createContext }).start(() =>
-  console.log(
-    `🚀 Server ready at: http://localhost:4000\n⭐️ See sample queries: http://pris.ly/e/ts/graphql-sdl-first#using-the-graphql-api`,
-  ),
+new ApolloServer({ schema, context: createContext }).listen(
+  { port: 4000 },
+  () =>
+    console.log(
+      `🚀 Server ready at: http://localhost:4000\n⭐️ See sample queries: http://pris.ly/e/ts/graphql-apollo-server#using-the-graphql-api`,
```

Comment:
I think the sample queries link is for the wrong example now :(

Reply:
Should be http://pris.ly/e/ts/graphql-sdl-first#using-the-graphql-api

Reply:
Nice catch, you've [got eyes like a hawk](https://en.pons.com/translate/german-english/Augen+wie+ein+Luchs+haben) (or lynx or whatever other animal that has great eyes) 😅

Fixed via: https://github.com/prisma/prisma-examples/pull/2201/files